README.md

```diff
@@ -9,22 +9,22 @@ multitasking, etc.
 
 ## Project Aims
 
-Objectives of this project for college evaluations :-
+Objectives of this project for college evaluations:-
 
-	1. Boot into 32-bit protected mode.
-	2. Interrupt Handling
-	3. Screen and keyboard drivers.
-	4. Basic _libc_ purely for the current requirements.
-	5. Filesystem.
-	6. Basic Memory Management.
-	7. Simple Shell.
-	8. Basic text editor.
-	9. User mode.
+    1. Boot into 32-bit protected mode.
+    2. Interrupt Handling
+    3. Screen and keyboard drivers.
+    4. Basic _libc_ purely for the current requirements.
+    5. Filesystem.
+    6. Basic Memory Management.
+    7. Simple Shell.
+    8. A basic text editor.
+    9. User mode.
 
-I would like to stretch my goals for :-
+I would like to stretch my goals for:-
```

Comment:
I think this 'for' doesn't look good and should be something like -
"I would like to stretch my goals to implement:-"
What do you think ??

Reply:
Sir, the latter looks more clear to me too, so I think we should modify it!